tests/Resources/ti.android.httpresponsecache.test.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Android.HttpResponseCache', function () {
```

Comment:
you can just do `describe.android('Titanium.Android.HttpResponseCache', function () {` and all the nested tests all be limited to android, so that you don't need to ad date filter to each `it`.

Reply:
done